app/routes/application.vue

```diff
@@ -0,0 +1,61 @@
+<template lang="html">
+  <div class="">
+    <nav class="panel">
+      <p class="panel-heading">
+        Sign Up For My Web App
+      </p>
+
+      <p class="control">
```

Comment:
Wrap the all of the `control` tags in a single `panel-block` to group them and add padding.